addons/skin.estuary/xml/Variables.xml

```diff
@@ -605,8 +607,17 @@
 		<value condition="ListItem.IsPlayable">icons/pvr/PVR-HasArchive.png</value>
 	</variable>
 	<variable name="SeasonEpisodeLabel">
-		<value condition="String.IsEmpty(ListItem.EpisodeName)">$INFO[ListItem.Season,S]$INFO[ListItem.Episode,E]</value>
-		<value>$INFO[ListItem.Season,S]$INFO[ListItem.Episode,E,: ]</value>
+		<value condition="!String.IsEmpty(ListItem.Season) | !String.IsEmpty(ListItem.Episode)">$INFO[ListItem.Season,S]$INFO[ListItem.Episode,E]$INFO[ListItem.EpisodePart,/]$INFO[ListItem.EpisodeName,: ]</value>
+		<value condition="!String.IsEmpty(VideoPlayer.EpisodeName)">$INFO[ListItem.EpisodeName]</value>
+	</variable>
+	<variable name="SeasonHighliteEpisodeLabel">
+		<value condition="!String.IsEmpty(ListItem.Season) | !String.IsEmpty(ListItem.Episode)">[COLOR grey]$INFO[ListItem.Season,S]$INFO[ListItem.Episode,E]$INFO[ListItem.EpisodePart,/][/COLOR][COLOR white]$INFO[ListItem.EpisodeName,: ][/COLOR]</value>
+		<value condition="!String.IsEmpty(VideoPlayer.EpisodeName)">[COLOR white]$INFO[ListItem.EpisodeName][/COLOR]</value>
+	</variable>
```

Comment:
I didn't like the name either I was explaining what it was doing.  I don't know much about skinning so thanks a lot for the tip and help.  I also made the semi-colon grey and took out these two conditions because (I think) the $INFO checks anyways
```
		<value condition="!String.IsEmpty(ListItem.EpisodeName)">$INFO[ListItem.EpisodeName]</value>
to 
		<value>$INFO[ListItem.EpisodeName]</value>
``
@ksooo do you want to to squash this to one commit?